app/assets/images/payment_icons/payflex.svg

```diff
@@ -0,0 +1 @@
0 01-.84.17h-.35a1.73 1.73 0 01-.37-.11 1.27 1.27 0 01-.36-.2 1.05 1.05 0 01-.28-.31v2.16h-.7V9.89zM18.83 11.92a.86.86 0 01-.35.14l-.44.08-.47.06a2.58 2.58 0 00-.42.12.76.76 0 00-.3.23.65.65 0 00-.12.41.49.49 0 00.07.28.41.41 0 00.18.19.71.71 0 00.25.11 1.55 1.55 0 00.31 0 1.74 1.74 0 00.59-.09 1.08 1.08 0 00.39-.24.81.81 0 00.23-.3.79.79 0 00.08-.31zM20 14.08a1 1 0 01-.5.1.61.61 0 01-.5-.18.67.67 0 01-.15-.49 1.6 1.6 0 01-.67.49 2.16 2.16 0 01-.81.15 2.35 2.35 0 01-.54-.06 1.33 1.33 0 01-.45-.21.94.94 0 01-.29-.36A1.13 1.13 0 0116 13a1.22 1.22 0 01.12-.58 1 1 0 01.32-.37 1.4 1.4 0 01.46-.22 4.37 4.37 0 01.52-.12l.54-.09a1.91 1.91 0 00.45-.08.69.69 0 00.31-.15.37.37 0 00.11-.3.64.64 0 00-.08-.36.49.49 0 00-.22-.21.76.76 0 00-.3-.1h-.33a1.46 1.46 0 00-.73.17.7.7 0 00-.32.63h-.69a1.6 1.6 0 01.16-.66 1.24 1.24 0 01.39-.43 1.76 1.76 0 01.56-.24 2.86 2.86 0 01.67-.08 5.09 5.09 0 01.56 0 1.61 1.61 0 01.5.19 1 1 0 01.37.35 1.18 1.18 0 01.14.6v2.15a2.93 2.93 0 000 .36c0 .07.09.11.2.11h.21zM22.08 14.65a9.7 9.7 0 01-.24.52 1.68 1.68 0 01-.26.35.88.88 0 01-.32.19 1.25 1.25 0 01-.4.06h-.25a.86.86 0 01-.23-.06v-.63l.2.07a.79.79 0 00.2 0 .61.61 0 00.36-.1.59.59 0 00.21-.29l.29-.7L20 9.89h.78L22 13.33l1.18-3.44h.73z"/><path class="cls-1" d="M24.07 10.5v-.61h.7v-.62a1 1 0 01.3-.78 1.25 1.25 0 01.86-.26h.45v.6H26a.7.7 0 00-.39.09.45.45 0 00-.14.38v.58h.82v.61h-.82v3.59h-.69V10.5zM26.93 8.29h.69v5.8h-.69zM31.34 11.13a1.15 1.15 0 00-.62-.63 1.12 1.12 0 00-.48-.1 1.06 1.06 0 00-.48.1 1 1 0 00-.37.25 1.19 1.19 0 00-.25.38 1.4 1.4 0 00-.1.46h2.41a1.41 1.41 0 00-.11-.46m.17 2.69a1.91 1.91 0 01-1.19.36 2.11 2.11 0 01-.87-.16 1.6 1.6 0 01-.62-.46 1.78 1.78 0 01-.38-.69 3.19 3.19 0 01-.14-.89 2.75 2.75 0 01.14-.88 2.38 2.38 0 01.41-.69 2 2 0 01.63-.45 1.86 1.86 0 01.79-.17 1.69 1.69 0 01.93.24 1.73 1.73 0 01.59.58 2.2 2.2 0 01.31.78 3.29 3.29 0 01.07.81H29a1.49 1.49 0 00.06.52 1.21 1.21 0 00.65.75 1.57 1.57 0 00.6.11 1.19 1.19 0 00.71-.2 1 1 0 00.37-.62h.69a1.7 1.7 0 01-.61 1.06"/><path class="cls-3" d="M6.53 11a4.85 4.85 0 000 2A4.85 4.85 0 019 14.29a3.28 3.28 0 010-4.66A4.93 4.93 0 016.53 11"/><path class="cls-3" d="M4.57 11a4.59 4.59 0 001 .1 4.73 4.73 0 001-.1 4.87 4.87 0 011.3-2.53 3.3 3.3 0 01-4.65 0A5 5 0 014.57 11"/><path class="cls-4" d="M6.53 12.94a4.73 4.73 0 00-1-.1 4.59 4.59 0 00-1 .1 5 5 0 01-1.35 2.51 3.3 3.3 0 014.65 0 4.87 4.87 0 01-1.34-2.51"/><path class="cls-4" d="M4.57 12.94a5.33 5.33 0 00.09-1 5.33 5.33 0 00-.09-1 5 5 0 01-2.52-1.31A3.26 3.26 0 013 12a3.28 3.28 0 01-1 2.33 4.88 4.88 0 012.52-1.35"/><path class="cls-5" d="M2.05 8.47a.82.82 0 000 1.16A5 5 0 004.57 11a5 5 0 00-1.35-2.53.83.83 0 00-1.17 0"/><path class="cls-2" d="M2.05 15.45a.83.83 0 001.17 0 5 5 0 001.35-2.51 4.88 4.88 0 00-2.52 1.35.82.82 0 000 1.16"/><path class="cls-5" d="M9 15.45a.83.83 0 01-1.17 0 4.87 4.87 0 01-1.34-2.51A4.85 4.85 0 019 14.29a.82.82 0 010 1.16"/><path class="cls-1" d="M9 8.47a.82.82 0 010 1.16A4.93 4.93 0 016.53 11a4.87 4.87 0 011.34-2.53.83.83 0 011.13 0"/></svg>
```

Comment:
- We're missing a white background. At the moment, it is transparent
- We're missing the grey border.
- Can we remove all classes and convert all `<styles>` into attributes? [![alt](https://screenshot.click/17-10-rbpix-794xc.png)](https://screenshot.click/17-10-rbpix-794xc.png)

Reply:
Thanks for the heads up Thomas. I've updated the SVG accordingly. Should be good to go. :)